pages/linux/arpspoof.md

```diff
@@ -0,0 +1,14 @@
+# arpspoof
+
+> Intercept packets on a switched LAN.
+> Redirects packets from a target host (or all hosts) on the LAN intended for another host on the LAN by forging ARP replies.
+> More information: <https://manned.org/arpspoof>.
+
+- Make a simple man-in-the-middle attack between two local hosts (run both at the same time, via `screen` or multiple terminal sessions):
```

Comment:
```suggestion
- Make a simple man-in-the-middle attack between the specified local hosts (`{{192.168.0.1}} {{192.168.0.2}}`):
```

Reply:
I'm not sure - this works only with 2 hosts (well, it might work with more, though I haven't tried that), and I think it's good to mention that both commands need to be run at the same time.

Reply:
The `{{token_syntax}}` and code blocks aren't normally used in descriptions?

Either way, I think the original description is pretty much ok? If I were to nitpick I'd say add `e.g.` before the `run both...` bit, but that's not necessary.